django/db/backends/postgresql/schema.py

```diff
@@ -367,3 +457,63 @@ def _is_collation_deterministic(self, collation_name):
             )
             row = cursor.fetchone()
             return row[0] if row else None
+
+    def _add_identity_sql(self, table, column):
+        return self.sql_add_identity % {
+            "table": self.quote_name(table),
+            "column": self.quote_name(column),
+        }
+
+    def _drop_identity_sql(self, table, column):
+        return self.sql_drop_indentity % {
```

Comment:
**Issue**: The method call uses 'sql_drop_indentity' which appears to be a typo and should be 'sql_drop_identity'. <br> **Fix**: Correct the typo in the method call from 'sql_drop_indentity' to 'sql_drop_identity'. <br> **Code Suggestion**: 
 ```
 -        return self.sql_drop_indentity % {
 +        return self.sql_drop_identity % {
 ```